triton/whisper/media/stats_summary_onnx_op14_single_batch.txt

```diff
@@ -0,0 +1,18 @@
+model name is decoder 
```

Comment:
Please put the model name `large-v2` in the filename 
`stats_summary_onnx_op14_single_batch.txt`